osp/core/ontology/oclass.py

```diff
@@ -147,6 +147,7 @@ def get_attribute_by_argname(self, name):
                     f"commandline tool to transform entity names to CamelCase."
                 )
                 return attribute
+        raise AttributeError(name)
```

Comment:
Makes a lot of sense, and actually we do not have a crash for example at cuds.py:809 because we are lucky that the action performed there also raises AttributeError.

Reply:
Yes exactly. But there will be a weird Attribute error in cuds.py:809.
Something like `None does not have attribute iri`.